src/WidgetUtils.js

```diff
@@ -100,7 +100,8 @@ export default class WidgetUtils {
      * @param {string} widgetId The ID of the widget to wait for
      * @param {boolean} add True to wait for the widget to be added,
      *     false to wait for it to be deleted.
-     * @returns {Promise} that resolves when the widget is available
+     * @returns {Promise} that resolves when the widget is the the
+     *     requested state according to the `add` param
```

Comment:
This doesn't make sense. "is in the" maybe?

Reply:
Oops.